Source/ThirdParty/easy_profiler/CMakeLists.txt

```diff
@@ -25,7 +27,13 @@ macro(easy_define_target_option TARGET SOURCE_OPTION TARGET_DEFINITION)
 endmacro()
 
 add_subdirectory(easy_profiler_core)
-add_subdirectory(profiler_gui)
+# ATOMIC BEGIN
+if (MSVC AND "${CMAKE_BUILD_TYPE}" STREQUAL "Debug")
+    message(STATUS "Profiler GUI can only be built in Release builds with MSVC compiler.")
```

Comment:
Does this cause the easy profiler cmake to look for debug Qt libs?  If so, maybe can be configured to use Release Qt libs regardless in easy profiler scripts?  If not, or another reason, lemme know.  I think this is acceptable, though would be good to have Profiler client available in debug builds, which could also be sorted later.